_clanky/využití-data-50-část-1.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+detail: true
+title: Využití Data50 -část 1
+ref: data50-1
+lang: cs
+image: ../attachments/články/využití-data-50/screen_vysledku.webp
+author: Michal Med
+date: 2021-01-18 03:14:15
+---
+První ze série článků o využití otevřených dat Zeměměřického úřadu se zabývá hledáním problému, který by data pomohla vyřešit a výběrem vhodných datových sad.
+
+# Využití open dat Zeměměřického úřadu pro povolování dočasných staveb - část 1
+
+<!--more-->
```

Comment:
@MichalMed to More musí být před prvním nadpisem, jinak se ten nadpis dostane i na home page a vypadá to takto:
![image](https://user-images.githubusercontent.com/718827/105146370-33ac7580-5b00-11eb-824f-0494ed96cfbb.png)